doc/news/changes.h

```diff
@@ -89,6 +95,11 @@ inconvenience this causes.
 
 
 <ol>
+  <li> New: Support for Abaqus mesh files has been added to the Grid_In class.
```

Comment:
The correct name of the class is actually `GridIn`. If you link to the name of the correct function `GridIn::read_abaqus()`, doxygen will even link there.